assets/default.scss

```diff
@@ -23,8 +23,8 @@ $spacing-unit:     1rem;
     --color-brand:      #2a7ae2;
 
     --color-grey:       #828282;
-    --color-grey-light: color-mix(in lch, var(--color-grey) var(--color-background), 40%);
-    --color-grey-dark: color-mix(in lch, var(--color-grey) var(--color-text), 25%);
+    --color-grey-light: color-mix(in lch, var(--color-grey), var(--color-background), 40%);
+    --color-grey-dark: color-mix(in lch, var(--color-grey), var(--color-text), 25%);
```

Comment:
This looks like a rebase error, this was certainly meant to be in the "Convert SASS variables to CSS custom properties" commit, not in "Color overhaul, search icon, and dark mode".

Reply:
Oh, that's actually a color overhaul one. But I guess the color-mix shouldn't have existed before that. I guess as long as we get a correct result (and a mostly correct one in prior commits) it's probably good enough. Without those commas, those lines mean nothing (as they get evaluated to nothing).